framework/src/modules/chain/synchronizer/block_synchronization_mechanism.js

```diff
@@ -396,28 +404,32 @@ class BlockSynchronizationMechanism extends BaseSynchronizer {
 				currentRound,
 			);
 
-			const blockIds = (await this.storage.entities.Block.get(
-				{
-					height_in: heightList,
-				},
-				{
-					sort: 'height:asc',
-					limit: heightList.length,
-				},
-			)).map(block => block.id);
+			const blockIds = (
```

Comment:
looks like the same formatting issue? if its fixing the formatting please ignore this comment.

Reply:
Yeah probably prettier got updated, it's fine.